ui/.prettierrc

```diff
@@ -3,5 +3,8 @@
   "jsxSingleQuote": true,
   "trailingComma": "es5",
   "useTabs": false,
-  "plugins": ["@ianvs/prettier-plugin-sort-imports"]
+  "plugins": [
```

Comment:
Commit message nit: We do not use a double-space to separate sentences in a paragraph.

Reply:
Sry, it still comes from flesh memory from time to time...fixed now.